packages/app/src/app/overmind/effects/vscode/Linter.ts

```diff
@@ -109,6 +109,7 @@ export class Linter {
       }
       if (kind[1] === 'less') return 'less';
       if (kind[1] === 'md') return 'markdown';
+      if (kind[1] === 'cljs') return 'clojurescript';
```

Comment:
This disables lint because we only lint certain values here and `closurescript` is not one of them. In practice I could have made this any other string and it would have worked too